scripts/tcl_commands/all.tcl

```diff
@@ -391,9 +403,9 @@ proc prep {args} {
         set ::env(LIB_SYNTH) $::env(TMP_DIR)/trimmed.lib
         trim_lib
 
-        set tracks_copy $::env(TMP_DIR)/tracks_copy.info
-        file copy -force $::env(TRACKS_INFO_FILE) $tracks_copy
-        set ::env(TRACKS_INFO_FILE) $tracks_copy
+        set tracks_processed $::env(TMP_DIR)/config.tracks
+        try_catch $::env(OPENROAD_BIN) -python scripts/new_tracks.py -i $::env(TRACKS_INFO_FILE) -o $tracks_processed
```

Comment:
I think `scripts/new_tracks.py` need to be replaced with `$::env(SCRIPTS_DIR)/new_tracks.py` because not all designs are run from openlane's top level directory

Reply:
Will tackle.